requirements/edx/github.in

```diff
@@ -73,5 +73,5 @@ git+https://github.com/edx/django-require.git@0c54adb167142383b26ea6b3edecc32118
 
 # Third Party XBlocks
 
-git+https://github.com/open-craft/xblock-poll@922cd36fb1c3cfe00b4ce03b19a13185d136447d#egg=xblock-poll==1.10.2
+git+https://github.com/open-craft/xblock-poll@fb40c93e2b55faa1d5abd6096da6287b7b42a6c6#egg=xblock-poll==1.12.0
```

Comment:
@gabor-boros, we shouldn't merge this until we merge https://github.com/open-craft/xblock-poll/pull/96.

Reply:
Good catch @Agrendalath  -- @gabor-boros , the commit hash also needs to be updated to the latest master commit once that merges.

I did some more testing on the sandbox to ensure that the Poll/Survey XBlocks can still download their survey results with this update, cf https://apps.pr29046.sandbox.opencraft.hosting/learning/course/course-v1:OpenCraft+Poll+2021_1/block-v1:OpenCraft+Poll+2021_1+type@sequential+block@acec7951b96545a1bf1bfee8694fe99e/block-v1:OpenCraft+Poll+2021_1+type@vertical+block@54ba90c96e9a405e84219d1d0cdf104a

Reply:
Thank you for the approvals @Agrendalath @pomegranited . The requirements are updated now. Please note that I used tag version instead of commit hash as we are tagging the library.